.rubocop.yml

```diff
@@ -271,3 +271,87 @@ Style/ExpandPathArguments:
 
 Style/FetchEnvVar:
   Enabled: true
+
+Style/FileRead:
+  Enabled: true
+
+Style/FileWrite:
+  Enabled: true
+
+Style/FloatDivision:
+  Enabled: true
+
+Style/For:
+  Enabled: true
+
+Style/FrozenStringLiteralComment:
+  Enabled: true
+
+Style/GlobalStdStream:
+  Enabled: true
+
+Style/GlobalVars:
+  Enabled: true
+
+Style/HashAsLastArrayItem:
+  Enabled: true
+
+Style/HashConversion:
+  Enabled: true
+
+# HashEachMethods does not guarantee the receiver is a Hash, so
+# this will not work since numerous puppet classes define #value
+# not #each_value
+Style/HashEachMethods:
+  Enabled: false
+
+Style/HashLikeCase:
+  Enabled: true
+
+Style/HashTransformKeys:
+  Enabled: true
+
+Style/HashTransformValues:
+  Enabled: true
+
+Style/IdenticalConditionalBranches:
+  Enabled: true
+
+Style/IfInsideElse:
+  Enabled: true
+
+Style/IfUnlessModifierOfIfUnless:
+  Enabled: true
+
+Style/IfWithBooleanLiteralBranches:
+  Enabled: true
+
+Style/IfWithSemicolon:
+  Enabled: true
```

Comment:
Looks like there's a todo for this one, do you want to resolve the source file or add the exclude here and remove it from todo?

```yaml
Exclude:
  - 'lib/puppet/pops/parser/evaluating_parser.rb'
```